llama_stack/providers/inline/post_training/huggingface/recipes/finetune_single_device_dpo.py

```diff
@@ -204,8 +205,8 @@ async def load_dataset(
     ) -> tuple[Dataset, Dataset, AutoTokenizer]:
         """Load and prepare the dataset for DPO training."""
         # Validate data config
-        if not config.data_config:
-            raise ValueError("DataConfig is required for DPO training")
+        if config.data_config is None:
+            raise MissingTrainingConfigError("DataConfig", "HuggingFace DPO Post-Training")
```

Comment:
Why do we have to repeat "HuggingFace DPO Post-Training", should we have a const instead?